src/components/tasks/card/card.module.scss

```diff
@@ -5,6 +5,8 @@
   flex-wrap: wrap;
   margin-bottom: 2em;
   cursor: pointer;
+  box-shadow: 0 0 15px -7px rgb(0, 0, 0 / 65%);
+  border-radius: 10px;
```

Comment:
What about hovering effect?

Reply:
My bad. Will fix that as well.

Reply:
![image](https://user-images.githubusercontent.com/10326494/113476121-10eef380-9497-11eb-95f5-26be9d6d8e45.png)

updated the same as above, is that ok?

Reply:
Yes, please push it